samples/net/rpl-node/README.rst

```diff
@@ -0,0 +1,72 @@
+.. _rpl-node-sample:
+
+RPL node
+###########
+
+Overview
+********
+
+A simple RPL node showing how to join in RPL mesh network.
```

Comment:
Change to:

     This sample builds a simple RPL node and shows how to join into an RPL mesh network.